tests/system_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# System test for the go-carbon -> zipper -> api stack.
+# Brings the systeem up, performs single write, and read.
+# Checks if data got in and out.
+#
+# Needs to be run from the root directory of the project.
+
+docker-compose build || { printf ">>> ERROR: Docker build failed"; exit 2; }
+docker-compose up -d || { printf ">>> ERROR: Launching containers failed"; exit 2; }
+
+# This is guestimate for the time when the system goes up and operational.
+# Doing this other ways is much more complex. May need to be increased on slow boxes.
+sleep 5
+
+echo "test.test.test 123456 $(date +%s)" | nc -q 1 localhost 2003 || { printf ">>> ERROR: Putting data in failed"; exit 2; }
+
+if curl "http://localhost:8081/render?target=test.test.test&format=json" | grep -q "123456"
```

Comment:
the sleep is unnecessary I believe. In case trigram index is not built, it falls back to file system lookup. A better way is to check if the trigram index is built or not, which should be almost instantaneous here (small dataset).

Reply:
@gksinghjsr Probably you are right, but I have added `2 sec` sleep just to be sure. This will be almost invisible vs the time it takes to do docker build and up.